master/buildbot/reporters/gerrit.py

```diff
@@ -397,6 +399,10 @@ def sendCodeReview(self, project, revision, result):
             return
 
         command = self._gerritCmd("review", "--project %s" % (project,))
+
+        if self.gerrit_notify is not None:
+            command.extend(["--notify %s" % str(self.gerrit_notify)])
```

Comment:
why do you use `.extend` for a single item?  use `.append`

please use `'` not `"`